baby-gru/src/utils/MoorhenMolecule.ts

```diff
@@ -1216,6 +1216,8 @@ export class MoorhenMolecule implements moorhen.Molecule {
         if (typeof selectionString === 'string') {
             this.hoverRepresentation.cid = selectionString
             await this.hoverRepresentation.redraw()
+            this.hoverRepresentation.drawSymmetry()
```

Comment:
I think we want this to be `this.hoverRepresentation.drawSymmetry(false)` otherwise it will pull the symmetry matrix from the cootworker every time we hover.

Reply:
Indeed! This is fine then.